app/web/test/setupTests.ts

```diff
@@ -45,9 +45,8 @@ global.crypto = {
   },
 };
 
-//sentry testing was causing OOM for some reason
-//const { testkit, sentryTransport } = sentryTestkit();
-//global.testKit = testkit;
+const { testkit } = sentryTestkit();
+global.testKit = testkit;
```

Comment:
Their next upgrade was focused mostly on performance and reducing bundle size so my guess was the OOM was happening due to that.